.github/workflows/slow.yaml

```
@@ -134,3 +134,52 @@ jobs:
         with:
           name: build-logs-macos
           path: btlayer-*.log
+
+  freebsd:
+    strategy:
+      matrix:
+        osversion:
+          - 14.1
+          - 13.3
+
+    runs-on: ubuntu-22.04
+    name: freebsd(${{ matrix.osversion }})
+
+    steps:
+      - name: Checkout Sources
+        uses: actions/checkout@v4
+
+      - name: Run test-builds
+        id: test-builds
+        uses: vmactions/freebsd-vm@v1
+        with:
+          usesh: true
+          release: ${{ matrix.osversion }}
+          prepare: |
+            pkg update
+            pkg install -y \
+              autoconf \
+              autoconf-archive \
+              automake \
+              bash \
+              cppunit \
+              gmake \
+              libltdl \
+              libtool \
+              m4 \
+              nettle \
+              pkgconf \
+              translate-toolkit
+
+          run: |
+            export MAKE=gmake
+            export CC=clang
+            export CXX=clang++
+            ./test-builds.sh layer-00-default layer-01-minimal layer-02-maximus
```

> unless we specify layers, the default also tests layer-04-noauth-everything which is inconsistent with what we do on Linux. macos has the same problem but it's out of scope to fix here.

A different fix may be needed and fixing this existing problem is outside _this_ PR scope (this PR does not need this fix to work and arguably becomes worse with it). Please keep doing what we have been doing for all non-matrix builds, before and after MacOS support was introduced:

```suggestion
            ./test-builds.sh
```

Let's discuss the best way to address this known testing inconsistency problem separately.

---

Sure, but let me share my puzzlement. This seems to be against the push for consistency you generally express, even when consistency makes things worse at least by some dimension

---

Thank you for addressing this change request.

> This seems to be against the push for consistency

Not specifying layers looks consistent with existing MacOS-testing code and inconsistent with Linux-testing code.  As far as overall layers handling is concerned, this FreeBSD-testing code is similar to MacOS code and dissimilar to Linux-testing code; MacOS and FreeBSD tests share the same approach and the same problem. Thus, the alleged inconsistency with Linux-testing code is irrelevant or less relevant here.

Besides consistency, specifying a custom ordered list of layers for a _single_ test-builds.sh call opens up a new/different  set of questions that we have not fully addressed when deciding what to test on Linux. We do not want to address those questions in _this_ PR AFAICT.